services/api/src/utils/emails.js

```diff
@@ -93,60 +113,34 @@ async function sendResetPassword({ to, token }) {
       subject: `Password Reset Request`,
     },
     {
-      html: template('reset-password.html', options),
-      text: template('reset-password.text', options),
-      options,
+      html: template({ template: 'reset-password.md', fields }),
     }
   );
 }
 
 async function sendInvite({ to, token, sender }) {
   await initialize();
-  const options = {
-    ...defaultOptions,
-    senderName: sender.name,
-    senderEmail: sender.email,
-    token,
-  };
-  return sendMail(
-    {
-      to,
-      subject: `${sender.name} has invited you to join ${defaultOptions.appName}`,
-    },
-    {
-      html: template('invite.html', options),
-      text: template('invite.text', options),
-      options,
-    }
-  );
-}
-
-async function sendInviteKnown({ to, token, sender }) {
-  await initialize();
-  const options = {
```

Comment:
love how much red I'm seeing ^^^^

Reply:
Yeah love this.

@kaareal Would it make sense to get rid of this `utils/emails` thing altogether? Instead have more generic functions?

So instead of `sendInvite()` we would have:

```javascript
const { sendEmailFromTemplate } = require('utils/emails');

await sendEmailFromTemplate('invite.md', {
  subject: '{{sender.name}} has invited you to join {{appName}}',
  to: invite.email,
  sender: user,
  token
})
```

Reply:
```
sendTemplatedMail({
    to: invite.email,
    subject: `${sender.name} has invited you to join {{appName}}`,
    template: 'invite.md',
    sender,
    token: getToken(invite),
  });
```